.github/workflows/ci-llama.yaml

```diff
@@ -8,6 +8,7 @@ name: Llama Benchmarking Tests
 
 on:
   workflow_dispatch:
+  pull_request:
   schedule:
     # Weekdays at 5:00 AM UTC = 10:00 PM PST.
     - cron: "0 5 * * 1-5"
```

Comment:
Why run on both `pull_request` and `schedule`? Usually we would run on `pull_request` and `push` _or_ `schedule`, not this combination.

See https://iree.dev/developers/general/github-actions/#workflow-triggers for conventions we use in IREE